src/app/states/editor/page.mdx

```diff
@@ -0,0 +1,62 @@
+export const metadata = {
+  title: 'editor',
+  description: 'An app state of the editor. This section explains how to fetch and manipulate the `editor` state.',
+}
+
+# editor
+
+An [app state](/guides/flux-architecture) of the editor.
+You can obtain the `editor` state and integrate it with React components.{{ className: 'lead' }}
+
+
+---
+
+## Data Structure
+
+<Row>
+  <Col>
+    <Properties>
+      <Property name="changed" type="boolean | undefined">
+        `true` if the note has been changed. 
+        The editor will save the note if `changed` is true when it is closed.
+        Otherwise it will be`undefined` if it has not been changed yet since the app launched.
+      </Property>
+      <Property name="needsUpdatingDate" type="boolean | undefined">
```

Comment:
```suggestion
      <Property name="needsUpdatingDate" type="boolean">
```
As far as I can tell, this property cannot be undefined
https://docs.inkdrop.app/reference/state-editor#needsupdatingdate-boolean